docs/src/app/components/pages/components/refresh-indicator.jsx

```diff
@@ -0,0 +1,82 @@
+let React = require('react');
+let { RefreshIndicator } = require('material-ui');
+let ComponentDoc = require('../../component-doc');
+
+
+let RefreshIndicatorPage = React.createClass({
+
+  render() {
+
+    let code =
+      '// Ready status \n' +
+      '<RefreshIndicator percentage={30} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={60} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={80} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={100} size={40} left={10} top={5} status="ready" />\n' +
+      '// Loading status \n' +
+      '<RefreshIndicator size={40} left={80} top={5} status="loading" />\n';
+
+    let componentInfo = [
+      {
+        name: 'Props',
+        infoArray: [
+          {
+            name: 'size',
+            type: 'number',
+            header: 'default: 40',
+            desc: 'Size in pixel.'
+          },
+          {
+            name: 'percentage',
+            type: 'number',
+            header: 'default: 0',
+            desc: 'The confirmation progress to fetch data. Max value is 100'
+          },
+          {
+            name: 'left',
+            type: 'number',
+            header: 'required',
+            desc: 'The left position of the indicator. Be care, the component is absolute positioned'
+          },
+          {
+            name: 'top',
+            type: 'number',
+            header: 'required',
+            desc: 'The top position of the indicator. Be care, the component is absolute positioned'
+          },
+          {
+            name: 'statu',
+            type: 'one of: ready, loading, hide',
+            header: 'default: hide',
+            desc: 'ready: means at the confirmation phase\nloading: means at data loading phase\nhide: means all done and hide the indicator'
```

Comment:
Please change the `desc` to: 'The display status of the indicator. If the status is "ready", the indicator will display the ready state arrow. If the status is "loading", it will display the loading progress indicator. If the status is "hide", the indicator will be hidden.'